resources/lib/model.py

```diff
@@ -401,6 +404,7 @@ def __init__(self, data: dict):
         self.id = panel.get("id")
         self.title: str = utils.format_long_episode_title(meta.get("season_title"), meta.get("episode_number"),
                                                           panel.get("title"))
+        self.name: str = panel.get("title", "")
```

Comment:
You've introduced the new `self.name`, which in most cases is a 1:1 duplication of `self.title` because of the `EpisodeData` item which adds the numbering to the title, right? 
I think it's a bit confusing what the real difference is between those two, if you don't know it. I'll see if I can come up with an idea for this.

Reply:
You add it throw the `unformatted_title` attribute.